### PR TITLE
chore(9630): update env var VERSION to use the branch name value

### PR DESCRIPTION
**Pull request by tatilepizs**
# Description

The idea is that when the automated tests are run, the images are not downloaded with every run. To do that, the env var `VERSION` would be updated to use the branch name value, and when the `ci` command is executed, it would ask if the `VERSION` is set, and if is not it would assign the value of the current branch and use the docker images that were built previously.

#9630 

# Code review checklist

- [ ] UI/UX backwards compatible: Test it works for the new design (enabled by default). And test it works in the old design, enable `can_view_old_navigation` permission to see the old design.
- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9630-update-version-var-to-use-branch-name/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9630-update-version-var-to-use-branch-name/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:9630-update-version-var-to-use-branch-name/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,12 +60,12 @@
...
-    "ci-integration-sentinel-k3d": "mocha --config tests/integration/.mocharc-sentinel-k3d.js",
-    "ci-webdriver-default": "wdio run ./tests/e2e/default/wdio.conf.js",
-    "ci-webdriver-default-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js",
+    "ci-integration-all": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-all.js",
+    "ci-integration-all-k3d": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-k3d.js",
+    "ci-integration-sentinel": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-sentinel.js",
+    "ci-integration-sentinel-k3d": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-sentinel-k3d.js",
+    "ci-webdriver-default": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && wdio run ./tests/e2e/default/wdio.conf.js",
+    "ci-webdriver-default-mobile": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} wdio run ./tests/e2e/default-mobile/wdio.conf.js",
```
Comment (dianabarsan) [inline package.json:68]: Are these even needed anymore? I think with the change of how we set versions locally, these are not needed.

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "dev-api": "./scripts/build/copy-static-files.sh && (npm run --prefix shared-libs/cht-datasource build-watch & npm run --prefix api run-watch)",
     "dev-sentinel": "npm run --prefix shared-libs/cht-datasource build-watch & npm run --prefix sentinel run-watch",
     "local-images": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && node scripts/build/cli localDockerComposeFiles",
+    "set-local-version": "export VERSION=${VERSION:-$(git branch --show-current)}",
```
Comment (dianabarsan) [inline package.json:32]: This is a duplicate of what the versions script is doing. Where should this be used? Is this needed?

File: package.json (hunk 1 above)
Comment (dianabarsan) [reply package.json:68]: Locally, with just the change in `scripts/build/versions.js` I can run the local command followed by the ci command, unchanged, and I get to use the correct images.

**Review by dianabarsan — CHANGES_REQUESTED** [review]
Nice work. Some questions inline.

File: package.json (hunk 1 above)
Comment (tatilepizs) [reply package.json:68]: Ohhh I thought that I needed to export the `VERSION` again to be able to use the correct images, but I just tested it and it worked perfectly, thanks Diana ⭐

File: package.json (hunk 2 above)
Comment (tatilepizs) [reply package.json:32]: It was a mistake, I was testing other options before, I will remove it, thanks!

**Review by dianabarsan — APPROVED** [review]
Sweet!

**Pull request merged**